src/ILCompiler.CppCodeGen/src/CppCodeGen/CppWriter.cs

```diff
@@ -1280,6 +1297,16 @@ private String GetCodeForReloc(Relocation reloc, DependencyNode node, NodeFactor
                     symbolNode.Target is GenericMethodsTemplateMap ||
                     symbolNode.Target is GenericTypesHashtableNode ||
                     symbolNode.Target is TypeMetadataMapNode ||
+                    symbolNode.Target is ReflectionInvokeMapNode ||
```

Comment:
This was originally a workaround for CppCodeGen limitations. It looks like we are able to emit pretty much anything now.

Do we still really need this? What are the `symbolNodes` that this is still filtering out?

Reply:
Now we didn't emit `GCStaticsRegion` (`ArrayOfEmbeddedPointersNode<GCStaticsNode>`), `ThreadStaticsRegion` (`ArrayOfEmbeddedDataNode<ThreadStaticsNode>`), `GCStaticEETypeNode`,  `EETypeOptionalFieldsNode`, `StructMarshallingStubMapNode`, `DelegateMarshallingStubMapNode`, `StackTraceMethodMappingNode`.
`StackTraceMethodMappingNode` cannot be emitted as it uses `IMAGE_REL_BASED_RELPTR32`.
Is it better to invert condition and check node that we want filter out?

Reply:
I would think so. Ideally, the exclusion would just go away eventually.

Reply:
Done. Thank you!